### PR TITLE
Implement Macros for Statistics Component

### PR DESCRIPTION
**Pull request by ckev07**
What:
- Statistics component was outdated and unrelated to the fitness / meal plans
- Mock data for fitness / meal plans were outdated and without the macros

How:
- Implement charts to display the nutritional macros fetched from mock fitness / meal plans
- Mock data updated to contain macros in both fitness (calories burned) / meal (calories, protein, carbs, fat) plans
- Update logic in fitness / meal plans component to account for addition of macros

### CODE REVIEW

**Review by jakji93 — APPROVED** [review]

File: client/src/components/Home/PlanView/ExistingMealSchedule.jsx [hunk 1]
```diff
@@ -38,18 +38,20 @@ export default function ExistingMealSchedule() {
       >
         {schedule && Object.keys(schedule).map((day, index) => {
           const dayPlan = schedule[day];
+          const mealPlan = { ...dayPlan };
+          delete mealPlan.nutrition_totals;
```
Comment (kennyjhcheng) [inline client/src/components/Home/PlanView/ExistingMealSchedule.jsx:42]: Why is this necessary? `mealPlan.nutrition_totals` isn't accessed in this component right

Comment (ckev07) [reply client/src/components/Home/PlanView/ExistingMealSchedule.jsx:42]: Based on the structure of the mealSchedule provided by @d-x-s , the `mealPlan` contains the meals (breakfast, snack1, lunch, etc.) as well as the nutrition_totals. Without the deletion on line 42, the meal plan in the FE would display the nutrition_totals, like `nutrition_totals: {Object : Object}`.

**Review by kennyjhcheng — APPROVED** [review]
